lib/src/map/options/map_options.dart

```diff
@@ -70,13 +97,13 @@ class MapOptions {
   /// Only use this if your map isn't built immediately (like inside FutureBuilder)
   /// and you need to access the controller as soon as the map is built.
   /// Otherwise you can use WidgetsBinding.instance.addPostFrameCallback
-  /// In initState to controll the map before the next frame.
+  /// In initState to control the map before the next frame.
```

Comment:
(For @JaffaKetchup) I need to fix the documentation here, as this method is now the recommended way to attach a controller if needed at initial build for reliability: see https://docs.fleaflet.dev/v/v7-beta/usage/programmatic-control/controller#usage-inside-initstate (link likely to change). Might as well do it in this PR, I'll make the change before merge.

Reply:
off topic, please do in a separate pr or in https://github.com/fleaflet/flutter_map/pull/1776